etna/models/nn/patchts.py

```diff
@@ -197,7 +197,8 @@ def step(self, batch: PatchTSBatch, *args, **kwargs):  # type: ignore
 
     def make_samples(self, df: pd.DataFrame, encoder_length: int, decoder_length: int) -> Iterator[dict]:
         """Make samples from segment DataFrame."""
-        values_real = df.select_dtypes(include=[np.number]).values
+        # TODO: is it ok that it passes target into values_real?
```

Comment:
Question to discuss

Reply:
Don't see any issues here, imho. It is used for teacher forcing https://github.com/etna-team/etna/blob/d229e7e77deb63fa3979b1587f82db1871f12bfc/etna/models/nn/patchts.py#L190

Reply:
It seems strange to me, because we already have target in `values_target`, and this model can't work with anything besides target.

Reply:
You could create an issue for more explicit behaviour